web/src/containers/admin/EditAccount.js

```diff
@@ -109,6 +117,43 @@ class EditAccount extends Component {
               </option>
             ))}
           </Select>
+
+          <div className="ds-l-row ds-u-padding-x--2">
+            <TextField
+              label="Password"
+              ariaLabel="Current password"
+              value="********"
+              disabled
+              size="medium"
+            />
+            <div className="ds-u-clearfix">
+              <div className="ds-c-label">&nbsp;</div>
```

Comment:
What's up with this empty label?

Reply:
It's to get the button vertically aligned with the textfield.  It seemed like the easiest way to me.  😬